challenge_serving_drinks/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(challenge_serving_drinks)
+
+## Find catkin macros and libraries
```

Comment:
Please cleanup all these outcommented lines. Use just what we actually use.

Reply:
See 48913250b433227ada9cd62bec88b001181e3ed5